### PR TITLE
release

### PR DESCRIPTION
**Pull request by aaronspring**
- or shall we take `0.7`? Its a big change as we upload different observations. EDIT: changed to `0.7.0`
- once we agree on the number here, I just do https://github.com/ecmwf-lab/climetlab-s2s-ai-challenge/releases, as release or pre-release?

### CODE REVIEW

**Comment by aaronspring** [comment]
website works: https://storage.ecmwf.europeanweather.cloud/s2s-ai-challenge/data/test-output-reference
file access forbidden: https://storage.ecmwf.europeanweather.cloud/s2s-ai-challenge/data/test-output-reference/t2m-20200312.nc

**Comment by aaronspring** [comment]
's2s-ai-challenge-observations' works still

**Comment by aaronspring** [comment]
`s2s-ai-challenge-test-output-reference` and `s2s-ai-challenge-training-output-reference` have forbidden urls

**Comment by floriankrb** [comment]
permissions granted, please try again.

**Comment by aaronspring** [comment]
works locally. should also here on CI

File: climetlab_s2s_ai_challenge/__init__.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 from . import extra  # noqa F401
 
 # note : this version number is the plugin version. It has nothing to do with the version number of the dataset
-__version__ = "0.6.7"
+__version__ = "0.6.8"
```
Comment (aaronspring) [inline climetlab_s2s_ai_challenge/__init__.py:15]:
```suggestion
__version__ = "0.7.0"
```

Comment (aaronspring) [reply climetlab_s2s_ai_challenge/__init__.py:15]: @floriankrb

**Comment by aaronspring** [comment]
preparing release: https://github.com/ecmwf-lab/climetlab-s2s-ai-challenge/releases/edit/untagged-29f041723d29ab87c968

**Pull request merged**